cartographer/common/time.cc

```diff
@@ -50,5 +52,13 @@ common::Duration FromMilliseconds(const int64 milliseconds) {
       std::chrono::milliseconds(milliseconds));
 }
 
+double GetThreadCpuTimeSeconds() {
+  struct timespec thread_cpu_time;
+  if (clock_gettime(CLOCK_THREAD_CPUTIME_ID, &thread_cpu_time) == -1) {
```

Comment:
There are several error codes, e.g. see: https://linux.die.net/man/3/clock_gettime
Checking "if (!clock_gettime() != 0)" is safer.

Reply:
"clock_gettime(), clock_settime() and clock_getres() return 0 for success, or -1 for failure (in which case errno is set appropriately)."

Reply:
+1. Let's check `!= 0`which is more common. Also: Use a `CHECK`? Right now we return bogus values on error.

Reply:
ok :) done.